src/main/java/io/confluent/connect/jdbc/util/SNSClient.java

```diff
@@ -60,13 +60,13 @@ protected AWSCredentialsProvider newCredentialsProvider(JdbcSourceConnectorConfi
     final String accessKeyId = config.getString(AWS_ACCESS_KEY_ID_CONFIG);
     final String secretKey = config.getPassword(AWS_SECRET_ACCESS_KEY_CONFIG).value();
     if (!accessKeyId.isEmpty() && !secretKey.isEmpty()) {
-      log.info("Returning new credentials provider using the access key id and "
+      log.debug("Returning new credentials provider using the access key id and "
           + "the secret access key that were directly supplied through the connector's "
           + "configuration");
       BasicAWSCredentials basicCredentials = new BasicAWSCredentials(accessKeyId, secretKey);
       return new AWSStaticCredentialsProvider(basicCredentials);
     }
-    log.info(
+    log.debug(
```

Comment:
Instead of changing log groups in each file whenever we need detailed logs, can we create some sort of wrapper and that wrapper can decide log level based on some environment variable ?

Reply:
This wrapper can be used for logging at critical places so if something goes wrong, we can just change env and redeploy pipeline and we will get detailed logs.

Reply:
Wouldn't it be the same thing? Currently, if you change the log level to debug, you would get the debug and info logs. You can change the log level to trace, if you want to see all the logs.

Reply:
No, in this case we changed the `log.info -> log.debug` in every file where we want to change the log level. Instead if we know some places which has critical logs we can use something like `customLog.log(...)` and this `customLog.log` can decide log level internally based on outside parameter. So it's implementation will be something like:

```
function log(...params) {
if(LOG_LEVEL_ENV === 'debug') {
 log.debug(...params);
} 

if (LOG_LEVEL_ENV === 'info') {
 log.info(...params)
}
// same for log.trace
}
```
This is not feasible for every place but can be used for critical logs for which we frequently change log level.